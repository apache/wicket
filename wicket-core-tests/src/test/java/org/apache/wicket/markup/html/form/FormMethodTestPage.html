<html>
<body>
	<form wicket:id="formpost"></form>
	<form wicket:id="formget"></form>
</body>
</html>
