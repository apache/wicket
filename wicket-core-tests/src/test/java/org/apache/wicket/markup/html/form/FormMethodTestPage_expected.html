<html>
<body>
	<form wicket:id="formpost" id="formpost1" method="post" action="./org.apache.wicket.markup.html.form.FormMethodTestPage?0-1.-formpost"><div id="formpost1_hf_0" hidden="" class="hidden-fields"></div></form>
	<form wicket:id="formget" id="formget2" method="get" action="./org.apache.wicket.markup.html.form.FormMethodTestPage"><div id="formget2_hf_0" hidden="" class="hidden-fields"><input type="hidden" name="0-1.-formget" value="" /></div></form>
</body>
</html>
